SmartDeviceLink/public/SDLChoiceSet.m

```diff
@@ -144,6 +147,14 @@ - (void)setHelpList:(nullable NSArray<SDLVRHelpItem *> *)helpList {
     }
 }
 
+- (NSTimeInterval)timeout {
+    if (_timeout == TimeoutDefault) {
+        return SDLChoiceSet.defaultTimeout;
+    }
+
+    return _timeout;
+}
```

Comment:
This doesn't look the same as `SDLAlertView` (neither does `+(NSTimeInterval)defaultTimeout`

Reply:
Updated to cap the timeout range between 5 and 100 seconds.